src/Core/Components/DataGrid/Columns/PropertyColumn.cs

```diff
@@ -96,5 +96,7 @@ protected override void OnParametersSet()
 	protected internal override void CellContent(RenderTreeBuilder builder, TGridItem item)
 		=> builder.AddContent(0, _cellTextFunc!(item));
 
+    protected internal override string? RawCellContent(TGridItem item)
+        => _cellTextFunc!(item);
```

Comment:
Why to use `!` because at the end of the `OnParametersSet` method, the `_cellTextFunc` can be null.
Maybe better to use `?` (same in `CellContent` method).

Reply:
Can use  `?` there because the you get error saying it can't convert `Func<TGridItem, string?>` to `bool`

Reply:
I think you have to use the Invoke method: `_cellTextFunc?.Invoke(item)`

Reply:
Yes, that works. Changing it now